ShenkinStore/Models/Product.cs

```diff
@@ -5,6 +5,9 @@ namespace ShenkinStore.Models
 {
     public class Product
     {
+        public bool sold { get; set; }
```

Comment:
This is a DB field? if so it's very not good, it will mix up between users

Reply:
I think it will be more reasonable to hold this data in the session or the cart and not in here

Reply:
Need to talk about